### PR TITLE
Changed MSA check delegations to return bool instead of Result

### PR DESCRIPTION
**Pull request by rlaferla**
Changed MSA check delegations to return `bool` instead of `Result`

### CODE REVIEW

File: pallets/msa/src/rpc/src/lib.rs [hunk 1]
```diff
@@ -78,14 +78,12 @@ where
 		let provider = Provider(provider_msa_id);
 
 		Ok(delegator_msa_ids
-			.par_iter()
+			.iter() // TODO: Change back to par_iter()
```
Comment (saraswatpuneet) [inline pallets/msa/src/rpc/src/lib.rs:81]: par_iter?

**Review by saraswatpuneet — APPROVED** [review]
One comment, lets change it to par_iter

File: pallets/msa/src/rpc/src/lib.rs (hunk 1 above)
Comment (rlaferla) [reply pallets/msa/src/rpc/src/lib.rs:81]: That's a future TODO.  We are intentionally reverting it to .iter().  @wilwade  will be making a more permanent fix which uses par_iter() again.

**Comment by rlaferla** [comment]
The iterator needs to be updated to a parallel iterator but there's a borrow issue when making this change that needs to be addressed.

File: pallets/msa/src/rpc/src/lib.rs [hunk 2]
```diff
@@ -78,14 +77,12 @@ where
 		let provider = Provider(provider_msa_id);
 
 		Ok(delegator_msa_ids
-			.par_iter()
+			.iter() // TODO: Change back to par_iter() which has borrow panic
```
Comment (enddynayn) [inline pallets/msa/src/rpc/src/lib.rs:80]: Please do not add TODO here. Instead, create tickets for them.

**Pull request merged**